internal/core/api/workspace.josh.hcl

```diff
@@ -5,10 +5,12 @@ interface "process" {
     output "job-id" "string" {}
   }
   method "stop" {
+    input "timeout-seconds" "*uint" {}
     output "job-id" "string" {}
   }
   # TODO: Optional method?
   method "restart" {
+    input "timeout-seconds" "*uint" {}
```

Comment:
Both the stop and stop-components methods are used by `exo stop`, but you added this for restart without changing `exo restart`.

Reply:
:+1: